packages/cli/api-importers/openapi/openapi-ir-to-fern/src/buildQueryParameter.ts

```diff
@@ -125,7 +125,11 @@ function getQueryParameterTypeReference({
                 }),
                 allowMultiple: true
             };
-        } else if (resolvedSchema.type === "oneOf" && resolvedSchema.value.type === "undiscriminated") {
+        } else if (
+            resolvedSchema.type === "oneOf" &&
+            resolvedSchema.value.type === "undiscriminated" &&
+            resolvedSchema.value.schemas.every((s) => s.type === "literal" && s.value.type === "string")
+        ) {
```

Comment:
nit: It'd be nice if there was a helper method for this, e.g. `isUndiscriminatedUnionOfLiterals` or something similar.